raceforcash/src/models/Horse.ts

```diff
@@ -0,0 +1,13 @@
+export interface Horse {
+    id: number;
+    name: string;
+    birthdate: Date;
+    breedId: number;
```

Comment:
server should send this as a string-breed

Reply:
Changed to string